### PR TITLE
Changes to the clamAV document

### PR DESCRIPTION
**Pull request by voroyam**
# New:

....
systemctl enable clamav-daemon.service
systemctl start clamav-daemon.service
....

----------------

Troubleshooting section

-------------

# Changed

Removed the occ command at the bottom, since in the container there is no sudo installed, so this command would fail. Also it's easier to tell the user to configure the clamAV instead of making him look up the possible values of the occ command.

### CODE REVIEW

File: modules/admin_manual/pages/appliance/configuration/clamav.adoc [hunk 1]
```diff
@@ -42,10 +42,19 @@ The installation should only take a few minutes.
 [[configure-owncloud-to-use-clamav]]
 == Configure ownCloud to Use ClamAV
 
+Start the ClamAV service:
+
+....
```
Comment (settermjd) [inline modules/admin_manual/pages/appliance/configuration/clamav.adoc]: Code blocks must be enclosed in `----` not `....`.

File: modules/admin_manual/pages/appliance/configuration/clamav.adoc [hunk 2]
```diff
@@ -86,5 +96,5 @@ sudo freshclam
 
 [WARNING]
 ====
-When the app is enabled — but is not configured or has an incorrect configuration — it will reject **all** uploads for the entire instance. To avoid this situation, use `{occ-command-example-prefix} config:app:set files_antivirus whatever_key value` to configure the app correctly.
+When the app is enabled — but is not configured or has an incorrect configuration — it will reject **all** uploads for the entire instance. To avoid this situation, make sure the ClamAV service is running and you have the execution mode correctly configured  in ownCloud.
```
Comment (settermjd) [inline modules/admin_manual/pages/appliance/configuration/clamav.adoc]: Please remove the double space between "configured" and "in".

File: modules/admin_manual/pages/appliance/configuration/clamav.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/clamav.adoc]: all the code blocks I have seen were in dots. The minuses break the code block

File: modules/admin_manual/pages/appliance/configuration/clamav.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/clamav.adoc]: removed

File: modules/admin_manual/pages/appliance/configuration/clamav.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/appliance/configuration/clamav.adoc]: Have a look at https://github.com/owncloud/docs/edit/master/modules/admin_manual/pages/appliance/configuration/clamav.adoc, line 55 - 58. That has a good example.

Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/clamav.adoc]: why is it different here then? https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code

**Pull request merged**